includes/customizer-functions.php

```diff
@@ -0,0 +1,63 @@
+<?php /**
+ * Hello Elementor customizer registration functions.
+ *
+ * @package HelloElementor
+ */
+
+/**
+ * Register Customizer controls which add Elementor deeplinks
+ *
+ * @return void
+ */
+add_action( 'customize_register', 'hello_customizer_register' );
+function hello_customizer_register( $wp_customize ) {
+	// Allow active/inactive via the Experiments
+	if ( ! hello_header_footer_experiment_active() ) return;
```

Comment:
phpcs

Reply:
```suggestion
	if ( ! hello_header_footer_experiment_active() ) {
		return;
	}
```

Reply:
Moved this to the registration action around the `require`